import-export-cli/integration/throttlePolicy_test.go

```diff
@@ -0,0 +1,154 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package integration
+
+import (
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/testutils"
+	"testing"
+)
+
+const (
+	ApplicationPolicy  = "application"
+	CustomPolicy       = "custom"
+	AdvancedPolicy     = "advanced"
+	SubscriptionPolicy = "subscription"
+)
+
+// Export an Application Throttling Policy from one environment and import to another environment
+func TestExportImportApplicationThrottlePolicy(t *testing.T) {
```

Comment:
We need to add test cases for update scenarios and error scenarios (such as exporting an invalid policy or importing a policy with incorrect/missing fields in the yaml file) as well.

Reply:
Added with ce216b1ed1373dcac938ffac86315e6386313099